StringSimilarity return bugfix (#28889)
demisto/content: Packs/CommonScripts/ReleaseNotes/1_12_14.md

#### Scripts

##### StringSimilarity

> - Fixed an issue with the script returning error instead of None when there is no similarity found

Reviewer: ```suggestion
- Fixed an issue where the the script errored instead of returning `None` when no similarity was found.
```
